daemon/provider/musicvideoinfo/spotifyprovider.py

```diff
@@ -173,3 +177,49 @@ def get_tracks_by_album_id(self, album_id):
                 break
 
         return tracks
+
+    def get_new_release_by_album_id(self, offset_count, limit):
```

Comment:
According to the above changed request, this function should be renamed to `get_new_release_albums()`. And the expected result should be the list of the album.